account_financial_report/models/account_group.py

```diff
@@ -0,0 +1,67 @@
+# © 2018 Forest and Biomass Romania SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountGroup(models.Model):
+    _inherit = "account.group"
+
+    group_child_ids = fields.One2many(
+        comodel_name="account.group", inverse_name="parent_id", string="Child Groups"
+    )
+    level = fields.Integer(compute="_compute_level", recursive=True)
+    account_ids = fields.One2many(
+        comodel_name="account.account", inverse_name="group_id", string="Accounts"
```

Comment:
We have a problem here: group_id on account.account was a stored field in v17, but it's NOT a stored field in v18. It seems to me that it is the cause of the message "2025-02-18 09:47:17,068 10499 ERROR o8_test1 odoo.osv.expression: Non-stored field account.account.group_id cannot be searched." ; you can see this message when you create a new company with a CoA for example. It breaks the tests in this PR https://github.com/OCA/l10n-france/pull/620 where I port a module that depend on account_financial_reporting